litmus-portal/frontend/src/views/AnalyticsDashboard/LitmusDashboard/WorkflowRunsBarChart/styles.ts

```diff
@@ -2,7 +2,7 @@ import { makeStyles } from '@material-ui/core';
 
 const useStyles = makeStyles(() => ({
   plot: {
-    width: '160%',
+    width: '164%',
```

Comment:
164???

Reply:
yes for now its required @arkajyotiMukherjee we are replacing this by litmus ui visx graphs.. this was required because of plotly's resize handler, its not taking full width without >100%

Reply:
is this plotly specific issue